website/source/docs/configuration/resources.html.md

```diff
@@ -83,6 +83,10 @@ include `create_before_destroy`. Referencing a resource that does not include
 ~> **NOTE on ignore\_changes:** Ignored attribute names can be matched by their
 name, not state ID. For example, if an `aws_route_table` has two routes defined
 and the `ignore_changes` list contains "route", both routes will be ignored.
+Additionally you can also use a single entry with a wildcard (e.g. `"*"`)
+which will match all attribute names. Using a partial string together with a
+wildcard (e.g. `"rout*"`) is **not** supported.
```

Comment:
Yes, I guess it would be good to inform users to manage their expectations. So added a [validation](https://github.com/hashicorp/terraform/pull/8599/files#diff-3baf47c64847a8fb8aaa8cc2e088513b) for this one.